### PR TITLE
Use a task to set up an additional HDD repository

### PR DESCRIPTION
**Pull request by poncovka**
Set up an additional HDD repository with a task.

### CODE REVIEW

File: pyanaconda/core/payload.py [hunk 1]
```diff
@@ -21,6 +21,25 @@
...
+    """
+    # Remove the prefix.
+    url = url.removeprefix("hd:")
+
+    # Split the specified URL.
+    components = url.split(":", maxsplit=1)
+
+    # Get two components.
+    # Add empty strings if necessary.
+    device, path = (components + [""] * 2)[:2]
```
Comment (M4rtinK) [inline pyanaconda/core/payload.py:38]: This seems a bit cryptic to me - can't we possibly do this in a more conventional way to make it more readable ?

**Review by M4rtinK — APPROVED** [review]
Looks good to me, thanks! :)

Just a small change suggestion.

File: pyanaconda/core/payload.py (hunk 1 above)
Comment (poncovka) [reply pyanaconda/core/payload.py:38]: Fixed at https://github.com/rhinstaller/anaconda/pull/4251.

**Pull request merged**